_dcp/mappings/data_jbossdeveloper_event/jbossdeveloper_event.json

```diff
@@ -0,0 +1,11 @@
+{
+  "jbossdeveloper_event": {
+    "properties": {
+      "sys_title": {
+        "type": "string",
+        "include_in_all": true,
+        "_boost": 1.5
+      }
```

Comment:
We should create mapping for both `start_date` and `end_date`. Add the following into `properties`:

    "start_date": { "type": "date" },
    "end_date": { "type": "date" }